packages/xchain-terra/src/client.ts

```diff
@@ -306,6 +307,36 @@ class Client extends BaseXChainClient implements XChainClient {
 
     return { from, to }
   }
+
+  /**
+   * Transaction to THORChain inbound address.
+   *
+   * @param {DepositParams} params The transaction options.
+   * @returns {TxHash} The transaction hash.
+   *
+   * @throws {"halted chain"} Thrown if chain is halted.
+   * @throws {"halted trading"} Thrown if trading is halted.
+   */
+  async deposit({ walletIndex = 0, asset = AssetLUNA, amount, memo }: DepositParams): Promise<TxHash> {
```

Comment:
Pls check latest `xchain-terra` - it's `transfer` method includes more parameters now

Reply:
Sorry, I only see params that I already pass
https://github.com/osabl/xchainjs-lib/blob/master/packages/xchain-terra/src/client.ts#L140
```suggestion
  async deposit({ walletIndex = 0, asset = AssetLUNA, amount, memo }: DepositParams): Promise<TxHash> {
```